pages/search/index.tsx

```diff
@@ -138,13 +138,13 @@ export default function Search({
             ) : null}
             {isLoading ? (
               <SkeletonLoader showImage={false} />
-            ) : drbResponse?.totalWorks > 0 ? (
+            ) : (
               <DRBContainer
                 drbResults={drbResults}
-                totalWorks={drbResponse.totalWorks}
+                totalWorks={drbResponse?.totalWorks || 0}
```

Comment:
Based on this and the tests, I think `DRBContainer` should itself be able to render the no results state as its default view. Right now, it depends on the parent passing in the value or `0` and the tests have it explicitly as `0`.

I think `DRBContainer` can set its `totalWorks` prop with an initial default value of `0`.